README.yaml

```diff
@@ -135,3 +135,27 @@ introduction: |-
   Find all documentation at: [atmos.tools](https://atmos.tools)
 
 contributors: []
+
+testing:
+  content: |-
```

Comment:
I think there's a misunderstanding how the `README.yaml` works. We run this though `gomplate` and apply this template: https://github.com/cloudposse/.github/blob/main/README.md.gotmpl

There's no section we look for called `content` or `testing`. 

Also, after updating the `README.yaml`, run `make readme` to rebuild it.

Note, we're going to do this with atmos soon, see:
- #934 

And the confusion here, shows we should have a JSON schema.